src/IgnoreList.cxx

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright The Music Player Daemon Project
+
+#include "IgnoreList.hxx"
+
+#include <cassert>
+#include <ranges>
+#include <algorithm>
+
+bool IgnoreListEntry::matches_record(const Record& record) const {
+	// The below logic would always return true if the entry is empty.
+	// This condition should never be true, as we don't push empty entries.
```

Comment:
Did you know that there's a special syntax for multi-line comments, and that `//` is the syntax for explicitly single-line comments?

Reply:
Apologies.